packages/editor/src/client/i18n.ts

```diff
@@ -26,8 +42,4 @@ i18n.use(LanguageDetector).use(initReactI18next).init({
   }
 })
 
-i18n.on('languageChanged', function (lng) {
```

Comment:
why did you remove this?

Reply:
@bishoymelek it's not needed anymore. As you can see on line 27. The locale will be passed to date-fns everytime it being used for a translation rendering.

@camilleory you can remove the code. With source control there is no need for commenting out code :)